infinario.py

```diff
@@ -286,12 +266,23 @@ def __init__(self, token, customer=None, target=None, silent=True, logger=None,
         :param logger: Instance of a logger used with silent == True
         :param transport: One of `NullTransport`, `SynchronousTransport`, `AsynchronousTransport`;
             consult their documentation as well
+        :param secret: Secret token of a project with analyses that should be exported
```

Comment:
It should be stated that secret is optional and that it can be also used for fetching segment of customer.